R/forecast-bbs-core.R

```diff
@@ -117,9 +117,7 @@ get_bbs_data <- function(start_yr, end_yr, min_num_yrs){
     bbs_results <- dbSendQuery(con, bbs_query)
     bbs_data <- dbFetch(bbs_results) %>%
       filter_species() %>%
-      filter(year >= start_yr, year <= end_yr) %>%
       group_by(site_id) %>%
-      filter(min(year) == start_yr, max(year) == end_yr, length(unique(year)) >= min_num_yrs) %>%
```

Comment:
Now that these have been removed it looks like maybe we don't need them as arguments to the function? In that case we should take them out of the function definition and any associated function calls.